web/packages/teleport/src/components/hooks/useUrlFiltering/useUrlFiltering.ts

```diff
@@ -39,7 +40,8 @@ export interface UrlFilteringState {
 }
 
 export function useUrlFiltering(
-  initialParams: Partial<ResourceFilter>
+  initialParams: Partial<Omit<ResourceFilter, 'includedResourceMode '>>,
```

Comment:
All of the keys in `ResourceFilter` are already optional, `Partial` doesn't seem necessary. There's also an extra space after `includedResourceMode`.

```suggestion
  initialParams: Omit<ResourceFilter, 'includedResourceMode'>,
```